ipykernel fails to build

**Describe the bug**
System update failed 


**To Reproduce**
Steps to reproduce the behavior:
1. Run
```
nix-channel --update && nixos-rebuild switch
```

**Expected behavior**
Error:
```
/nix/store/iwz5gar7p27ic2nqd8i4p0lfhsa2g8dg-vscode-1.41.1/lib/vscode/resources/app/extensions/ms-vscode.node-debug/dist/termi
nateProcess.sh: interpreter directive changed from "/bin/bash" to "/nix/store/j78ycszn9vgn0khl0lcpbd087x2mmr66-bash-4.4-p23/b
in/bash"

manual-combined.xml validates
/nix/store/iwz5gar7p27ic2nqd8i4p0lfhsa2g8dg-vscode-1.41.1/lib/vscode/resources/app/out/vs/base/node/cpuUsage.sh: interpreter 
directive changed from "/bin/bash" to "/nix/store/j78ycszn9vgn0khl0lcpbd087x2mmr66-bash-4.4-p23/bin/bash"
/nix/store/iwz5gar7p27ic2nqd8i4p0lfhsa2g8dg-vscode-1.41.1/lib/vscode/resources/app/out/vs/base/node/terminateProcess.sh: inte
rpreter directive changed from "/bin/bash" to "/nix/store/j78ycszn9vgn0khl0lcpbd087x2mmr66-bash-4.4-p23/bin/bash"
collected 86 items / 3 errors / 83 selected                                    
/nix/store/iwz5gar7p27ic2nqd8i4p0lfhsa2g8dg-vscode-1.41.1/lib/vscode/resources/app/out/vs/base/node/ps.sh: interpreter direct
ive changed from "/bin/sh" to "/nix/store/j78ycszn9vgn0khl0lcpbd087x2mmr66-bash-4.4-p23/bin/sh"

==================================== ERRORS ====================================
____________ ERROR collecting ipykernel/tests/test_embed_kernel.py _____________
ImportError while importing test module '/build/ipykernel-5.1.3/ipykernel/tests/test_embed_kernel.py'.
Hint: make sure your test modules/packages have valid Python names.
Traceback:
ipykernel/tests/test_embed_kernel.py:13: in <module>
    from flaky import flaky
E   ModuleNotFoundError: No module named 'flaky'
_______________ ERROR collecting ipykernel/tests/test_kernel.py ________________
ImportError while importing test module '/build/ipykernel-5.1.3/ipykernel/tests/test_kernel.py'.
Hint: make sure your test modules/packages have valid Python names.
Traceback:
ipykernel/tests/test_kernel.py:14: in <module>
    from flaky import flaky
E   ModuleNotFoundError: No module named 'flaky'
____________ ERROR collecting ipykernel/tests/test_start_kernel.py _____________
ImportError while importing test module '/build/ipykernel-5.1.3/ipykernel/tests/test_start_kernel.py'.
Hint: make sure your test modules/packages have valid Python names.
Traceback:
ipykernel/tests/test_start_kernel.py:1: in <module>
    from .test_embed_kernel import setup_kernel
ipykernel/tests/test_embed_kernel.py:13: in <module>
    from flaky import flaky
E   ModuleNotFoundError: No module named 'flaky'
=============================== warnings summary ===============================
ipykernel/iostream.py:14
  /build/ipykernel-5.1.3/ipykernel/iostream.py:14: DeprecationWarning: the imp module is deprecated in favour of importlib; s
ee the module's documentation for alternative uses
    from imp import lock_held as import_lock_held

ipykernel/pickleutil.py:8
  /build/ipykernel-5.1.3/ipykernel/pickleutil.py:8: DeprecationWarning: ipykernel.pickleutil is deprecated. It has moved to i
pyparallel.
    warnings.warn("ipykernel.pickleutil is deprecated. It has moved to ipyparallel.", DeprecationWarning)

ipykernel/codeutil.py:17
  /build/ipykernel-5.1.3/ipykernel/codeutil.py:17: DeprecationWarning: ipykernel.codeutil is deprecated since IPykernel 4.3.1
. It has moved to ipyparallel.serialize
    warnings.warn("ipykernel.codeutil is deprecated since IPykernel 4.3.1. It has moved to ipyparallel.serialize", Deprecatio
nWarning)

ipykernel/serialize.py:7
  /build/ipykernel-5.1.3/ipykernel/serialize.py:7: DeprecationWarning: ipykernel.serialize is deprecated. It has moved to ipy
parallel.serialize
    warnings.warn("ipykernel.serialize is deprecated. It has moved to ipyparallel.serialize", DeprecationWarning)

-- Docs: https://docs.pytest.org/en/latest/warnings.html
!!!!!!!!!!!!!!!!!!! Interrupted: 3 errors during collection !!!!!!!!!!!!!!!!!!!!
======================== 4 warnings, 3 errors in 1.24s =========================
/nix/store/iwz5gar7p27ic2nqd8i4p0lfhsa2g8dg-vscode-1.41.1/lib/vscode/bin/code: interpreter directive changed from "/usr/bin/e
nv bash" to "/nix/store/j78ycszn9vgn0khl0lcpbd087x2mmr66-bash-4.4-p23/bin/bash"
checking for references to /build/ in /nix/store/iwz5gar7p27ic2nqd8i4p0lfhsa2g8dg-vscode-1.41.1...
builder for '/nix/store/ahmk0xz2svp4agk8vi5j0kpkvcrm8fn1-python3.7-ipykernel-5.1.3.drv' failed with exit code 2
cannot build derivation '/nix/store/g1wj6gcdwaw17sc21i6qn220csz8l086-python3-3.7.6-env.drv': 1 dependencies couldn't be built
cannot build derivation '/nix/store/5cx23h0fnq5gwpsb8jax3ns6a0qcykzk-user-environment.drv': 1 dependencies couldn't be built
cannot build derivation '/nix/store/yzl04qhw4qblwp2a1cb2q16vnzijc864-etc.drv': 1 dependencies couldn't be built
cannot build derivation '/nix/store/v323vbrrlic4djd99sjc40rxfikqikkv-nixos-system-desktop-nixos-19.09.1764.2d9454702e5.drv': 
1 dependencies couldn't be built
error: build of '/nix/store/v323vbrrlic4djd99sjc40rxfikqikkv-nixos-system-desktop-nixos-19.09.1764.2d9454702e5.drv' failed
```
**Additional context**
Usual system update

**Metadata**
```
 - system: `"x86_64-linux"`
 - host os: `Linux 5.4.7, NixOS, 19.09.1748.ad1e1af5ad3 (Loris)`
 - multi-user?: `yes`
 - sandbox: `yes`
 - version: `nix-env (Nix) 2.3.1`
 - channels(root): `"home-manager-19.09, nixos-19.09.1764.2d9454702e5, unstable-20.03pre207998.e0470e11c7a"`
 - nixpkgs: `/nix/var/nix/profiles/per-user/root/channels/nixos`
```

--- a/pkgs/development/python-modules/ipykernel/default.nix
+++ b/pkgs/development/python-modules/ipykernel/default.nix
@@ -33,7 +33,7 @@ buildPythonPackage rec {
     })
   ];
 
-  checkInputs = [ pytestCheckHook nose ];
+  checkInputs = [ pytestCheckHook nose flaky ];
   dontUseSetuptoolsCheck = true;
   preCheck = ''
     export HOME=$(mktemp -d)

